### ISSUE TITLE
How to delete attributes tag?

### ISSUE THREAD

**Issue by oovm** [y21/tl#29]
How to remove certain tags from Attributes?

**Comment by y21** [comment]
This looks to be one of the methods that exist in the underlying `HashMap` struct ([`InlineHashMap::remove`](https://docs.rs/tl/latest/tl/inline/hashmap/struct.InlineHashMap.html#method.remove)), but are missing in the wrapping `Attributes` struct.
Seems like something that could be added easily. Also, while we are at it, we should add `Attributes::contains` as well.

**Comment by oovm** [comment]
any progress?

**Comment by y21** [comment]
Can you elaborate on what you mean by "remove the value but keep the key"?
Would this be the same as `*attribute.get_mut().unwrap() = None;` ?

**Comment by oovm** [comment]
```html
<div contenteditable="true">
  This text can be edited by the user.
</div>
```

- remove_key

```rust
attributes.remove_key("contenteditable")
```

```html
<div>
  This text can be edited by the user.
</div>
```

- remove_value

```rust
attributes.remove_value("contenteditable")
```

```html
<div contenteditable>
  This text can be edited by the user.
</div>
```

**Comment by y21** [comment]
i'll get started on a pr, although i would like `Attributes::remove_key()` to be named `Attributes::remove()` because `remove_key` confused me a little bit.

**Comment by y21** [comment]
published 0.7.2 with new methods on `Attributes`:
- [`Attributes::contains(key)`](https://docs.rs/tl/latest/tl/struct.Attributes.html#method.contains)
- [`Attributes::remove(key)`](https://docs.rs/tl/latest/tl/struct.Attributes.html#method.remove)
- [`Attributes::remove_value(key)`](https://docs.rs/tl/latest/tl/struct.Attributes.html#method.remove_value)
